src/hotspot/share/gc/shenandoah/shenandoahControlThread.hpp

```diff
@@ -173,11 +173,11 @@ class ShenandoahControlThread: public ConcurrentGCThread {
   void prepare_for_graceful_shutdown();
   bool in_graceful_shutdown();
 
-  void service_concurrent_normal_cycle(const ShenandoahHeap* heap,
+  void service_concurrent_normal_cycle(ShenandoahHeap* heap,
```

Comment:
Isn't this delta already there in another PR? I'd rebase this PR wrt that as parent, or land that and refresh this so as to avoid presenting the same diffs in two different PRs where possible.